lib/index.js

```diff
@@ -78,6 +81,8 @@ export default class Pageres<DestValue: string> extends EventEmitter {
 		this.options = Object.assign({}, options);
 		this.options.filename = this.options.filename || '<%= url %>-<%= size %><%= crop %>';
 		this.options.format = this.options.format || 'png';
+		this.options.increment = this.options.increment || false;
+		this.options.defaultDelimiter = this.options.defaultDelimiter ? this.options.defaultDelimiter : false;
```

Comment:
I think you can drop this line. Later on in the code, you check for `this.options.defaultDelimiter` which means that if it's `undefined`, it will already be evaluated as `false`.